docs/source/notebooks/summary.rst

```diff
@@ -0,0 +1,2 @@
+List of Notebook Examples
```

Comment:
Might not need this but thought it might be worth including one page that has quick one-line summaries of each notebook example so people don't have to click into each individual one